### PR TITLE
Fix broken tests batch 2

### PR DESCRIPTION
**Pull request by thomtrp**
As title

### CODE REVIEW

File: packages/twenty-front/src/hooks/__tests__/usePageChangeEffectNavigateLocation.test.ts [hunk 1]
```diff
@@ -134,7 +134,7 @@ const testCases = [
   { loc: AppPath.InviteTeam, isLoggedIn: true, subscriptionStatus: SubscriptionStatus.Active, onboardingStatus: OnboardingStatus.Completed, res: defaultHomePagePath },
 
   { loc: AppPath.PlanRequired, isLoggedIn: true, subscriptionStatus: undefined, onboardingStatus: OnboardingStatus.PlanRequired, res: undefined },
-  { loc: AppPath.PlanRequired, isLoggedIn: true, subscriptionStatus: SubscriptionStatus.Canceled, onboardingStatus: OnboardingStatus.Completed, res: undefined },
+  { loc: AppPath.PlanRequired, isLoggedIn: true, subscriptionStatus: SubscriptionStatus.Canceled, onboardingStatus: OnboardingStatus.Completed, res: defaultHomePagePath },
```
Comment (thomtrp) [inline packages/twenty-front/src/hooks/__tests__/usePageChangeEffectNavigateLocation.test.ts:137]: to check with @charlesBochet and @martmull if it still makes sense

Comment (FelixMalfait) [reply packages/twenty-front/src/hooks/__tests__/usePageChangeEffectNavigateLocation.test.ts:137]: I think we should redirect to '/settings/billing' in that case? Or 'undefined' seems like an acceptable option too. This is very theoretical, probably doesn't matter much, but one of the two options would be best imo

**Review by FelixMalfait — APPROVED** [review]
Great thanks for taking care of this! One small comment but can be merged after that!

File: packages/twenty-front/src/hooks/__tests__/usePageChangeEffectNavigateLocation.test.ts (hunk 1 above)
Comment (martmull) [reply packages/twenty-front/src/hooks/__tests__/usePageChangeEffectNavigateLocation.test.ts]: no, we should not redirect in order to be able to pay for a new subscription, so undefined is the proper expected response

**Pull request merged**